doc/modules/ROOT/pages/index.adoc

```diff
@@ -5,8 +5,16 @@ You can clone this repository, work on it and create your guide, and then push t
 ////
 
 :github-address: [github-repo-address-of-your-guide]
+
 :templates-url: https://raw.githubusercontent.com/hazelcast-guides/adoc-templates/master
 
+// Use this relative url if you are going to publish the guide on the guides site.
+// Note that this url will not work locally and raise asciidoctor errors.
+// So, complete the guide with the above url and set the below one just before 
+// publishing on the guides site.
+//
+// :templates-url: templates:ROOT:page$/
```

Comment:
When this url is used, it will be fine on the website but the index.adoc here in the repo will have dummy links pointing nowhere. So, reading the doc from the repository will not be a good choice any more.